src/Services/SwaggerService.php

```diff
@@ -325,6 +372,15 @@ protected function parseResponse($response)
                 $produce
             );
         }
+
+        $this->saveResponseSchema($content, $code);
+
+        if (is_array($this->item['responses'][$code])) {
+            $action = Str::ucfirst($this->getActionName($this->uri));
+            $definition = "#/definitions/{$this->method}{$action}{$code}ResponseObject";
```

Comment:
let's move difinitionaName to the separate variable and use it here and pass it to the `saveResponseSchema` method